apps/google-analytics-4/frontend/src/components/main-app/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+import tokens from '@contentful/f36-tokens';
+
+const ACCESSIBILITY_LABEL = 'Analytics line chart';
```

Comment:
Is there a reason to make this into a var instead of an inline string value in the template below?

Reply:
I have previously followed practices that generally suggest all static constants should be defined above the component, to separate them from more interactive code. Personally, I find it helps with readability within JSX, but am certainly open to including them within the component itself! Do you tend to have a preference or practice with that?

Reply:
Not particularly. My preference for variables in the template is more for repeatability. I.E. if a template string is used more than once, it deserves a variable. Your approach is also good, I wouldn't say either one is right or wrong, only personal preference. 

No need to change anything here 👍